ffmpeg/_utils.py

```diff
@@ -41,3 +42,8 @@ def escape_chars(text, chars):
     for ch in chars:
         text = text.replace(ch, '\\' + ch)
     return text
+
+
+def sort_ordereddict(d):
+    for key in sorted(d.keys()):
+        d[key] = d.pop(key)
```

Comment:
This moves the key to the end of the dict:
- https://docs.python.org/3.6/library/collections.html#collections.OrderedDict